tests/52user-directory/01public.pl

```diff
@@ -384,12 +400,25 @@
       my ( $user ) = @_;
 
       my $room_id;
+      my $searching_user;
 
       my $displayname = generate_random_displayname();
       my $second_displayname = generate_random_displayname();
+      my $searching_displayname = generate_random_displayname();
 
       matrix_set_displayname( $user, $displayname )
       ->then( sub {
+
+         matrix_create_user_on_server( $user->http,
```

Comment:
I'm not quite sure why we are creating a user that is never used here?

Reply:
they have to join the room so that the dir is populated

Reply:
they don't seem to join a room?